providers/dns/auroradns/auroradns.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/edeckers/auroradnsclient/records"
 	"github.com/edeckers/auroradnsclient/zones"
 	"github.com/xenolf/lego/acme"
+	"github.com/xenolf/lego/providers/dns/envvar"
```

Comment:
Doesn't make sense to me to have this included in the `providers/dns/envvar` package I'd almost say instead to create a `platform/config/env` might be a better fit that would allow better extension. (To perhaps add a dynamic/database backed configuration package)